source/documentation/rules-b.html.md.erb

```diff
@@ -0,0 +1,668 @@
+---
+title: NCTS Phase 5 Technical Interface Specification
+weight: 5
+description: Software developers, designers, product owners or business analysts. Integrate your software with the ERMIS service
+---
+#Rules B
+Based on document version 5.15.0-v0.10 and issue date 04/03/2022
+<%= partial 'documentation/partials/rulesintro' %>
+##B1000
+
+<b>Functional Description</b>
+
+Export followed by Transit & Groupage:<br/><br/>During the Transitional Period, in case of “Export Followed by Transit” with multiple export declarations covered by one standard transit declaration (i.e. not a pre-lodged transit declaration), the D.G. PREVIOUS DOCUMENT at CONSIGNMENT ITEM level can include the MRN of the related Export declaration (maximum one Export MRN included per one Consignment item).
+
+<b>Technical Description</b>
+
+N/A<br/>
+
+##B1003
+
+<b>Functional Description</b>
+
+In case of 'Road Transport' if the road vehicle (and/or its road trailers) are different to those declared in the initial declaration message, then they will be reported as discrepancies and not as new transport means.
+
```

Comment:
Grammar/style issues - the text in parentheses complicates "they".

Suggestion:
In the case of 'Road Transport', if a road vehicle and/or its trailers are different from those declared in the initial declaration message, they will be reported as discrepancies and not as new transport means.

Reply:
This is a direct copy from the DDNTA so we're not able to change the wording.